test/object-store/c_api/c_api.cpp

```diff
@@ -4663,9 +4663,10 @@ TEST_CASE("app: flx-sync basic tests", "[c_api][flx][sync]") {
                 {
                     using namespace std::chrono_literals;
                     std::unique_lock<std::mutex> lock{m_mutex};
-                    m_cv.wait_for(lock, 300ms, [this]() {
+                    bool completed_within_time_limit = m_cv.wait_for(lock, 5s, [this]() {
```

Comment:
Mmm, the failure should be related to some async call that either never got called back or did not complete on time. Extending the timeout should suffice, but probably the problem is another, for example how slow was the response from the baas server?

Reply:
yes, I suspect that 300ms is not enough time for a network call when the server is busy